exo-setup/src/app-setup.ls

```diff
@@ -27,15 +27,16 @@ class AppSetup extends EventEmitter
         @services.push do
             role: service-role
             location: service-data.location
-            docker-image: service-data.docker_image
+            docker-image: service-data['docker-image']
 
     @_setup-services ~>
-      @_get-dependencies-docker-config!
-      @_get-service-docker-config!
-      @_render-docker-compose!
-      @_setup-docker-images (exit-code) ~>
-        | exit-code => @write 'setup failed'; process.exit exit-code
-        @write 'setup complete'
+      @_get-dependencies-docker-config (err) ~>
+        | err => @write 'setup failed'; process.exit
```

Comment:
This doesn't actually call `process.exit`. Also the default exit code is zero (success). We should probably supply 1. Also thoughts on adding a test for this case where we would exit on this line? (If you agree, try to add the test before fixing this so we can see it go from red to green)